nostr/relay.py

```diff
@@ -57,7 +68,8 @@ def add_subscription(self, id, filters: Filters):
 
     def close_subscription(self, id: str) -> None:
         with self.lock:
-            self.subscriptions.pop(id)
+            if id in self.subscriptions:
+                self.subscriptions.pop(id)
```

Comment:
This is the proper syntax: 

```suggestion
            self.subscriptions.pop(id, None)
```